tests/Unit/ParallelAlgorithms/LinearSolver/Gmres/Test_ElementActions.cpp

```diff
@@ -111,15 +120,20 @@ SPECTRE_TEST_CASE("Unit.ParallelAlgorithms.LinearSolver.Gmres.ElementActions",
     INFO("InitializeElement");
     CHECK(get_tag(LinearSolver::Tags::IterationId<DummyOptionsGroup>{}) ==
           std::numeric_limits<size_t>::max());
-    tmpl::for_each<
-        tmpl::list<initial_fields_tag, operator_applied_to_fields_tag,
-                   operand_tag, operator_applied_to_operand_tag,
-                   orthogonalization_iteration_id_tag, basis_history_tag>>(
+    tmpl::for_each<tmpl::list<
+        initial_fields_tag, operator_applied_to_fields_tag, operand_tag,
+        std::conditional_t<Preconditioned,
```

Comment:
`tmpl::conditional_t`

Reply:
What’s the difference between the two?

Reply:
consistency throughout the code base and unless the `std::conditional_t` uses compiler intrinsics, `tmpl:conditional_t` will be kinder to the compiler